teamC/JeongDayoung/app.js

```diff
@@ -0,0 +1,40 @@
+dotenv.config();
+// Build-in package
+const http = require("http");
+
+// 3rd-party package
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+const app = express();
+app.use(express.json());
+app.use(morgan('dev'));
+
```

Comment:
cors에 관한 middleware도 추가 해주세요-!
```suggestion
app.use(cors());
```